src/kvilib/config/KviBuildInfo.cpp

```diff
@@ -134,11 +138,7 @@ namespace KviBuildInfo
 
 	QString buildSystemVersion()
 	{
-#ifdef COMPILE_ON_WINDOWS
```

Comment:
While I'm fine with the idea in general... What's the point of a function which returns an empty string? Please remove the function

Reply:
dropped it.